common/src/main/java/com/wynntils/models/statuseffects/type/StatusEffect.java

```diff
@@ -28,8 +33,49 @@ public StatusEffect(StyledText name, StyledText modifier, StyledText displayedTi
                 name,
                 StyledText.fromString(" "),
                 displayedTime);
+
+        if( modifier == StyledText.EMPTY ){
```

Comment:
I am not sure if spotless catches this or not, but try to run it.. Our code base does not use this style of conditions, please remove the spaces from before and after the conditions.

Reply:
I ran spotlessCheck and it passed, so I thought it'd be ok